sdk/test/UnitTests/Tools/CustomResponses.cs

```diff
@@ -31,9 +31,9 @@ public static class CustomResponses
     {
 #if NET45
         public static void SetResponse(
-            AmazonServiceClient client, string content, string requestId, bool isOK)
+            AmazonServiceClient client, string content, string requestId, string s3ExtendedRequestId, bool isOK)
```

Comment:
Using default parameters here with default value as null or true (in case of isOK) can eliminate the need for passing a bunch of null values and a true in most of the test cases calling this method. This will be a bit cleaner. If you can do that as part of this PR it would be great! :)

Reply:
Just changed